tests/kernel/sched/schedule_api/src/user_api.c

```diff
@@ -78,4 +86,266 @@ void test_user_k_is_preempt(void)
 
 	twp = thread_was_preempt;
 	zassert_true(twp, "unexpected return value");
+	k_thread_abort(tid);
+}
+
+/* userspace negative test: take NULL as input to verify the api will invoke fatal exception */
+#ifdef CONFIG_USERSPACE
+static void thread_suspend_init_null(void *p1, void *p2, void *p3)
+{
+	ztest_set_fault_valid(true);
+	k_thread_suspend(NULL);
+
+	/* should not go here */
+	ztest_test_fail();
+}
+
+/**
+ * @brief Test k_thread_suspend() API
+ *
+ * @details Create a thread and set k_thread_suspend() input para to NULL
+ * will trigger fatal error.
```

Comment:
Does "para" should be "param"?

Reply:
hi, enjia. Thank you for you careful review. I've changed you mentioned above.